redash/handlers/users.py

```diff
@@ -133,3 +136,23 @@ def post(self, user_id):
         return user.to_dict(with_api_key=is_admin_or_owner(user_id))
 
 
+class UserDisableResource(BaseResource):
+    @require_admin
+    def post(self, user_id):
+        user = models.User.get_by_id_and_org(user_id, self.current_org)
+        # admin cannot disable self; current user is an admin (`@require_admin`)
+        # so just check user id
+        if user.id == current_user.id:
+            abort(400, message="This user cannot be disabled.")
```

Comment:
Let's change the message to: `You cannot disable your own account. Please ask another admin to do this for you.`.